### PR TITLE
Change Helm broker url prefix list

### PR DESCRIPTION
**Pull request by adamwalach**
**Description**
Changes proposed in this pull request:

- definition of list of supported prefixes was moved to helm chart

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/5102

### CODE REVIEW

File: add-ons/src/constants.ts [hunk 1]
```diff
@@ -76,14 +73,12 @@ export const MODAL = {
   DELETE_MODAL_TITLE: 'Delete',
 };
 
-const HELP_URL_FIELD_DEVELOPMENT_MODE =
-  'The URL must be unique for a given configuration and start with the http(s) protocol or one of the following prefixes: git::, github.com/, bitbucket.org/';
-const HELP_URL_FIELD_NOT_DEVELOPMENT_MODE =
-  'The URL must be unique for a given configuration and start with the https protocol or one of the following prefixes: git::, github.com/, bitbucket.org/';
+const HELP_URL_FIELDS_LIST =
+  "'" + HELM_BROKER_REPO_URL_PREFIXES.join("', '") + "'";
```
Comment (parostatkiem) [inline add-ons/src/constants.ts]:
```suggestion
  `'${HELM_BROKER_REPO_URL_PREFIXES.join(`', '`)}'`;
```

File: add-ons/src/constants.ts [hunk 2]
```diff
@@ -76,14 +73,11 @@ export const MODAL = {
...
-const HELP_URL_FIELD_DEVELOPMENT_MODE =
-  'The URL must be unique for a given configuration and start with the http(s) protocol or one of the following prefixes: git::, github.com/, bitbucket.org/';
-const HELP_URL_FIELD_NOT_DEVELOPMENT_MODE =
-  'The URL must be unique for a given configuration and start with the https protocol or one of the following prefixes: git::, github.com/, bitbucket.org/';
+const HELP_URL_FIELDS_LIST = `'${HELM_BROKER_REPO_URL_PREFIXES.join(`', '`)}'`;
 export const HELP = {
-  URL_FIELD: HELM_BROKER_IS_DEVELOPMENT_MODE
-    ? HELP_URL_FIELD_DEVELOPMENT_MODE
-    : HELP_URL_FIELD_NOT_DEVELOPMENT_MODE,
+  URL_FIELD:
```
Comment (magicmatatjahu) [inline add-ons/src/constants.ts]:
```js
URL_FIELD: `The URL must be unique for a given configuration and start with one of the following prefixes: ${HELP_URL_FIELDS_LIST}`
```

File: add-ons/src/constants.ts [hunk 3]
```diff
@@ -110,12 +104,9 @@ export const ERRORS = {
   },
   URL: {
     ALREADY_EXISTS: 'This URL already exists in the configuration.',
-    STARTS_WITH_HTTP: {
-      DEVELOPMENT_MODE:
-        'URL must start with the http(s) protocol or one of the following prefixes: git::, github.com/, bitbucket.org/',
-      PRODUCTION_MODE:
-        'URL must start with the https protocol or one of the following prefixes: git::, github.com/, bitbucket.org/',
-    },
+    WRONG_PREFIX:
```
Comment (magicmatatjahu) [inline add-ons/src/constants.ts]:
```js
WRONG_PREFIX: `URL must start with one of the following prefixes: ${HELP_URL_FIELDS_LIST}`
```

File: add-ons/src/constants.ts (hunk 2 above)
Comment (adamwalach) [reply add-ons/src/constants.ts]: fixed

File: add-ons/src/constants.ts (hunk 3 above)
Comment (adamwalach) [reply add-ons/src/constants.ts]: fixed

**Pull request merged**